### ISSUE TITLE
ModalFooter does not allow children

### ISSUE THREAD

**Issue by gregorylegarec** [cozy/cozy-ui#388]
It could be nice to be able to show any content in `ModalFooter`, and not only buttons for actions.

For example :
```jsx
<ModalFooter>
  Sticky Footer content, could be text or any other kind of stuff
</ModalFooter>
```

**Comment by GoOz** [comment]
Yeah that's partly why I renamed this component, I figured it would be nice to have anything else than buttons but I ~didn't had the time~ totally forgot to add this feature when I refactored the modal.
